src/app/styles/index.scss

```diff
@@ -6,6 +6,8 @@ html {
 
 body {
   font-family: $font-primary;
+  // https://github.com/rolling-scopes/site/issues/590
```

Comment:
I'd prefer not to leave a comment here, but let's wait for other team members' opinions

Reply:
i agree

Reply:
Okay, will change it

Reply:
Done 🥳